app/controllers/portal_controller.rb

```diff
@@ -112,6 +112,16 @@ def annotations
     end
   end
 
+  def gallery
+    @gallery = Gallery.published.joins(:images).
```

Comment:
I don't think this is really the way https://europeana.atlassian.net/browse/EC-1901 intends the galleries to be included on item pages. I thought we were going to load in any galleries that are included in annotations?
So this method should be for 'galleries' not 'gallery' and the retrieve all the galleries via the annotations.